recipes/code-server/meta.yaml

```
@@ -0,0 +1,55 @@
+{% set name = "code-server" %}
+{% set version = "2.1692-vsc1.39.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url: https://github.com/cdr/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 73292bbdc857443210d8d7b4a641683a31d1039aef69c374480000b544a7307b
+  patches:
+    - node-binary.patch
+
+build:
+  number: 0
+  binary_relocation: False  # [osx]
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - python =2.7
+    - nodejs =10.*
+    - yarn
+    - pkg-config  # [linux]
+  host:
+    - nodejs =10.*
+    - libxkbfile  # [linux]
+    - xorg-libx11  # [linux]
+    - icu
+    - libuv
+    - openssl
+    - zlib
+  run:
+    - nodejs =10.*
```

I haven't seen this before, so I'm not sure it works.
```suggestion
    - nodejs 10.*
```